tests/test_helper.py

```diff
@@ -0,0 +1,23 @@
+from tests import JinaTestCase
+from jina.helper import cached_property
+
+
+class MyTestCase(JinaTestCase):
```

Comment:
can you put a more descriptive name to the `TestCase` class? And shouldn't it be under `tests/unit`?